app/config/auth.js

```diff
@@ -48,7 +49,8 @@ const authConfig = {
     hostname: process.env.RPA_HOST_NAME,
     getPersonSummaryUrl: process.env.RPA_GET_PERSON_SUMMARY_URL,
     getOrganisationPermissionsUrl: process.env.RPA_GET_ORGANISATION_PERMISSIONS_URL,
-    getOrganisationUrl: process.env.RPA_GET_ORGANISATION_URL
+    getOrganisationUrl: process.env.RPA_GET_ORGANISATION_URL,
```

Comment:
RPA_GET_ORGANISATION_URL needs to be added to _config-map.yaml_ and a new value added to _values.yaml_, this will ensure it works when deployed to an environment that isn't local